giraffe/src/types/index.ts

```diff
@@ -255,6 +255,7 @@ export interface MosaicLayerConfig {
   strokeOpacity?: number
   fillOpacity?: number
   yColumnType: ColumnType
+  yTicks: string[]
```

Comment:
Amazing. now typescript should know if config is of type mosaic, yTicks must be string.